### PR TITLE
Add chart for createdb job, bump trillian version.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

## Description of the change
Add a chart inside trillian for createdb which will initialize a Trillian DB schema.

## Checklist

- [ x ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ x ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

**Review by priyawadhwa — APPROVED** [review]
nice!!

**Review by k4leung4 — COMMENTED** [review]
to confirm, there is no harm rerunning the createdb job when there is one already right?
or should we add a flag to toggle the createdb job?

**Comment by vaikas** [comment]
> to confirm, there is no harm rerunning the createdb job when there is one already right? or should we add a flag to toggle the createdb job?

Correct, it will simply check the tables and if they exist, it doesn't do anything.
https://github.com/sigstore/scaffolding/blob/main/cmd/trillian/createdb/main.go

But, to be extra super duper sure, I'll add a flag for toggling it!

File: charts/trillian/templates/createdb/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{- if .Values.createdb.serviceAccount.create }}
```
Comment (k4leung4) [inline charts/trillian/templates/createdb/serviceaccount.yaml:1]: nit: wrap this with {{- if .Values.createdb.enabled }} as well

**Review by k4leung4 — APPROVED** [review]

**Review by k4leung4 — APPROVED** [review]
thank you

**Pull request merged**